packages/fuels-abigen-macro/tests/harness.rs

```diff
@@ -1435,3 +1436,21 @@ async fn test_wallet_balance_api() {
     assert!(balances.contains_key(&expected_key));
     assert_eq!(*balances.get(&expected_key).unwrap(), 231)
 }
+
+#[tokio::test]
+async fn test_require_sway() {
```

Comment:
Merge the two tests together, and call it `test_running_script` for instance.

Reply:
IMO you should put  the test in the `script.rs` file itself, this is more a unit test than an harness test.